relay-server/src/actors/processor.rs

```diff
@@ -2237,6 +2238,43 @@ impl EnvelopeProcessorService {
         Ok(())
     }
 
+    #[cfg(feature = "processing")]
+    fn extract_spans(&self, state: &mut ProcessEnvelopeState) {
+        // For now, drop any spans submitted by the SDK.
+        state.managed_envelope.retain_items(|item| match item.ty() {
+            ItemType::Span => ItemAction::DropSilently,
+            _ => ItemAction::Keep,
+        });
```

Comment:
I don't know how much load standalone spans in envelopes will represent to consider moving this logic to PoPs.

Reply:
There should not be any, so for now this is just a precaution.